channel.go

```diff
@@ -59,3 +72,13 @@ func (c *Channel) addUser(user User) {
 func (c *Channel) Users() []User {
 	return c.users
 }
+
+// IsValidChannelName checks the length and format of a string and returns
+// true if the string is valid
+func IsValidChannelName(name string) bool {
+	if len(name) < 50 && ValidChannelNameRgx.MatchString(name) {
```

Comment:
Will 
```go
len(name) < 50 && ValidChannelNameRgx.MatchString(name)
```
evaluate to a boolean? If it will, I would simply return the result of that instead of using the conditional here to return the true/false